code/iaas/external-handler/src/main/java/io/cattle/platform/extension/dynamic/dao/ExternalHandlerDao.java

```diff
@@ -1,11 +1,13 @@
 package io.cattle.platform.extension.dynamic.dao;
 
 import io.cattle.platform.core.model.ExternalHandler;
+import io.cattle.platform.extension.dynamic.data.ExternalHandlerData;
 
 import java.util.List;
 
 public interface ExternalHandlerDao {
 
-    List<? extends ExternalHandler> getExternalHandler(String eventName);
+    List<? extends ExternalHandler> getExternalHandler(String processName);
```

Comment:
Not used but is the default method to retrieve an ExternalHandler.  I don't think there's any harm to keep it there for now in case we can make use of it later.